fix/convertor.go

```diff
@@ -0,0 +1,37 @@
+package fix
+
+import (
+	"sync"
+
+	"github.com/b2broker/simplefix-go/fix/buffer"
+)
+
+type MessageByteConverter struct {
+	pool sync.Pool
+}
+
+func NewMessageByteConverter(bufferPoolPreAllocationCount, bufferSize int) *MessageByteConverter {
+	b := &MessageByteConverter{
+		pool: sync.Pool{
+			New: func() interface{} {
+				return buffer.NewMessageByteBuffers(bufferSize)
+			},
+		},
+	}
+	for i := 0; i < bufferPoolPreAllocationCount; i++ {
+		b.pool.Put(buffer.NewMessageByteBuffers(bufferSize))
```

Comment:
Looks like unused pool warmup, because actually pool does not behave like cache. It dynamically manages objects inside and they will be collected by GC.